mlflow/langchain/utils/chat.py

```diff
@@ -252,12 +264,14 @@ def should_transform_requst_json_for_chat(lc_model):
     return True
 
 
-def transform_request_json_for_chat_if_necessary(request_json, lc_model):
+def transform_request_json_for_chat_if_necessary(request_json: dict, lc_model):
```

Comment:
Sorry for the native question, do we intentionally remove type signature here (like [this](https://python.langchain.com/api_reference/core/language_models/langchain_core.language_models.chat_models.BaseChatModel.html)?) to support multiple versions?

Reply:
Emm I think type hint here doesn't help so much, and it introduces extra imports (while langchain's import path can change across different versions :))